src/components/ManageHouses.vue

```diff
@@ -35,35 +39,33 @@
     :closable="false"
     :dismissableMask="true"
   >
-    <EditHouseForm
+    <ManagingHouseForm
       :cooperationId="$props.cooperationId"
-      :propsHouseData="this.house"
-      @house-saved="displayModalForEditHouse = false"
-      @cancel-editing="displayModalForEditHouse = false"
-    ></EditHouseForm
-  ></Dialog>
+      :houseData="this.house"
+      :isEditHouse="true"
+      @cancel-managing="displayModalForEditHouse = false"
+    ></ManagingHouseForm>
+  </Dialog>
 </template>
 
 <script lang="ts">
 import { defineComponent } from 'vue';
 import Button from 'primevue/button';
 import Dialog from 'primevue/dialog';
-import EditHouseForm from '@/components/EditHouseForm.vue';
 import ListOfHouses from '@/components/ListOfHouses.vue';
-import AddHouseForm from '@/components/AddHouseForm.vue';
 import { HouseModel } from '@/shared/models/house.model';
 import { AddressModel } from '@/shared/models/address.model';
 import { HousesActionsEnum } from '@/store/houses/types';
 import { StoreModuleEnum } from '@/store/types';
+import ManagingHouseForm from '@/components/ManagingHouseForm.vue';
```

Comment:
ManageHouseForm would be better name, in my opinion.

Reply:
Got it